Bottom: 17df4f629429d338272f1084504d270fb07881ad
Top:    496dae100a1d3b59f011f293f6188a25cdc51146
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2013-02-11 00:43:13 +0000

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 70cc17f..d413f57 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2459,7 +2459,12 @@ IsPolygonInPolygon (PolygonType *P1, PolygonType *P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
